lib/hooks/typography/typography.treat.ts

```diff
@@ -1,5 +1,5 @@
 import mapValues from 'lodash/mapValues';
-import { style, styleMap, ClassRef } from 'sku/treat';
+import { style, styleMap, ClassRef, styleTree } from 'sku/treat';
```

Comment:
Apologies in advance.

```suggestion
import { style, styleMap, styleTree, ClassRef } from 'sku/treat';
```

Reply:
Thanks for not getting mad at me.

Reply:
I saw this and wanted to change it. You calling it out was the tipping point 😂